core/src/apps/ethereum/layout.py

```diff
@@ -258,35 +284,6 @@ async def confirm_typed_value(
         )
 
 
-def format_ethereum_amount(
```

Comment:
I am surprised that moving this function did not break unittests - in `core/tests/test_apps.ethereum.layout.py`  we are importing `format_ethereum_amount` still from this file.

(As it is imported here also, maybe it works, but we should change the import to `from helpers`)

Reply:
Yep, it picked up the import from `layout.py`. I fixed it so that it's imported directly.
https://github.com/trezor/trezor-firmware/pull/3534/commits/b3eb30b259e2034a10e613d2f20c7761538a91f7